### DISCUSSION TITLE
Problème d'installation

### DISCUSSION THREAD

**Discussion by loran48a** [dbarzin/deming#106] [Q&A] [answered]
Bonjour MrBarzin,
Déjà utilisateur de Mercator je m'interesse aujourd'hui à Deming. Merci beaucoup  pour votre partage ! Malheureusement je me trouve devant un message d' erreur dans le début de l'installation suite à la ligne de commande 'composer install' , je n' arrive pas en trouver la cause malgré mes recherches :( Pouvez-vous m'aider svp ? Voilà une image qui sera sûrement plus parlante que mes explications, Merci par avance.
![ErreurComposer](https://github.com/user-attachments/assets/7305abbd-e85f-4c36-be35-114022ed9a1a)

**Comment by dbarzin** [comment, marked as answer]
Il est possible que j'aie comité du code invalide vers le 23 juillet.
Tout semble correct dans la version actuelle.
